packages/metrics/README.md

```diff
@@ -0,0 +1,53 @@
+# @loopback/metrics
+
+This module contains a component to report metrics to Prometheus.
+
+## Installation
+
+```sh
+npm install --save @loopback/metrics
+```
+
+## Basic use
+
+The component should be loaded in the constructor of your custom Application
+class.
+
+Start by importing the component class:
+
+```ts
+import {MetricsComponent} from '@loopback/metrics';
+```
+
+In the constructor, add the component to your application:
+
+```ts
+this.component(MetricsComponent);
+```
+
+By default, Metrics route is mounted at `/metrics`. This path can be customized
```

Comment:
How about access control?

Reply:
It's similar as how we expose openapi specs. Let's worry about that later.

There are a few things to consider:
1. pull vs push - prometheus prefers `pull`
2. use a different rest endpoint (host/port)
3. ACL